sdk/rust/oak/src/io/mod.rs

```diff
@@ -51,18 +51,35 @@ pub fn node_create<T: Encodable + Decodable>(
 
 /// Creates a node and corresponding inbound channel of the same type, for nodes that are
 /// instantiated via the [`crate::entrypoint_command_handler`] macro.
-pub fn entrypoint_node_create<T: crate::WasmEntrypoint + crate::CommandHandler>(
+pub fn entrypoint_node_create<T: crate::WasmEntrypoint>(
     name: &str,
     label: &Label,
     wasm_module_name: &str,
-) -> Result<Sender<T::Command>, OakStatus>
-where
-    T::Command: Encodable + Decodable,
```

Comment:
Why don't we need `Encodable + Decodable` ?

Reply:
The bounds are now already in `WasmEntrypoint`.